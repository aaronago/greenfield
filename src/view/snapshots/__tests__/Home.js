
import {createSnapshot} from '../createSnapshot'

import homeData from './Home.json'

createSnapshot('Home view loads and renders as expected', homeData.payload)
