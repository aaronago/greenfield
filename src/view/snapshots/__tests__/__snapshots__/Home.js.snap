// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home view loads and renders as expected 1`] = `
<div
  className="sc-htpNat dEQUjC"
>
  <header
    className="sc-bwzfXH bsTHfU"
  >
    <img
      alt="logo"
      className="sc-bdVaJa cruMCr"
      src="logo.svg"
    />
    <h2>
      Welcome to React
    </h2>
  </header>
  <p>
    Head to the 
    <a
      href="/about"
      onClick={[Function]}
      onMouseDown={false}
      onTouchStart={false}
      target={undefined}
    >
      About Page
    </a>
    .
  </p>
</div>
`;
